run_create_sns.py

```diff
@@ -31,6 +31,14 @@ def run_create_sns_topic(name, settings):
         cmd += ['--notification-endpoint', settings['EMAIL']]
         aws_cli.run(cmd)
 
+    if 'PAGERDUTYAPPKEY' in settings and settings['PAGERDUTYAPPKEY']:
```

Comment:
Gendo Cloudwatch에 별도의 topic을 주기 위해 변경을 진행했습니다.
변경된 topic의 코드
https://github.com/HardBoiledSmith/magi/pull/816/files#diff-1294f9c4722a0792f3afe1bdc1c6d75b7339dff1fbd7264bf78867746126178fR246

Reply:
@GoldieKMG // 이렇게 코드를 줄여볼수도 있을것 같습니다. :) 
```suggestion
    if settings.get('PAGERDUTYAPPKEY'):
```